cocos/bindings/dop/BufferAllocator.h

```diff
@@ -86,9 +84,10 @@ class CC_DLL BufferAllocator final : public cc::Object {
     static cc::vector<BufferAllocator *> pools;
     static constexpr uint                BUFFER_MASK = ~(1 << 30);
 
-    cc::map<uint, Object *>  _buffers;
-    cc::map<uint, uint8_t *> _caches;
-    PoolType                 _type = PoolType::UNKNOWN;
+    cc::map<uint, Object *> _buffers;
+    cc::vector<uint8_t *>   _array;
```

Comment:
Should use a more meaningful name, such as _cachedBufferDataPointers.

Reply:
> Should use a more meaningful name, such as _cachedBufferDataPointers.

ok